tests/fixtures/pretty.sql

```diff
@@ -264,22 +264,3 @@ CREATE TABLE "t_customer_account" (
   "account_no" VARCHAR(100)
 );
 
-CREATE TABLE "t_customer_account" (
```

Comment:
why was this deleted?

Reply:
The `ENGINE` property is MySQL specific, so this test was moved to the MySQL dialect tests.